publishable/database/seeds/SettingsTableSeeder.php

```diff
@@ -13,120 +13,120 @@ public function run()
         $setting = $this->findSetting('site.title');
         if (!$setting->exists) {
             $setting->fill([
-                'display_name' => 'Site Title',
-                'value'        => 'Site Title',
+                'display_name' => __('voyager.seeders.settings.site.title'),
+                'value'        => __('voyager.seeders.settings.site.title'),
                 'details'      => '',
                 'type'         => 'text',
                 'order'        => 1,
-                'group'        => 'Site',
+                'group'        => __('voyager.seeders.settings.site.group_name'),
```

Comment:
Group should not be translated.  Group is used as part of the "key" when referencing the setting.

Reply:
https://github.com/the-control-group/voyager/blob/1.0/src/Http/Controllers/VoyagerSettingsController.php#L19-L29

What about this?

Reply:
Those are the settings that dont have a key.
They will be displayed, and only that, in a group named `General` or its translation.
 Imagine the following in german:
```php
$setting = $this->findSetting('site.title');
if (!$setting->exists) {
    $setting->fill([
        'display_name' => __('voyager.seeders.settings.site.title'),
        'value'        => __('voyager.seeders.settings.site.title'),
        'details'      => '',
        'type'         => 'text',
        'order'        => 1,
        'group'        => 'Seite',
    ])->save();
}
```
This will save the setting `site.title` in the group `Seite`.
You can call `setting('site.title')` **until** you edit/save settings, then you will get a key called `seite.site.title` and you will have to call it like this.